test/integration/test_create_dss_docker_image.py

```diff
@@ -0,0 +1,68 @@
+import docker
+import logging
+import pytest
+import requests
+import time
+
+from exasol.ds.sandbox.lib.dss_docker import DssDockerImage
+from datetime import datetime
+
+
+@pytest.fixture(scope="session")
+def dss_docker_container():
+    timestamp = f'{datetime.now().timestamp():.0f}'
+    testee = DssDockerImage(
+        "my-repo/dss-test-image",
+        version=f"{timestamp}",
+        publish=False,
+        log_level=logging.INFO,
+    )
+    print(
+        "\n- Using"
+        f' Docker container {testee.container_name}'
+        f' with image {testee.image_name}'
+    )
+    testee.create()
+    client = docker.from_env()
+    mapped_ports = {'8888/tcp': 8888}
+    container = client.containers.create(
+        image=testee.image_name,
+        name=testee.container_name,
+        command="sleep infinity",
+        detach=True,
+        ports=mapped_ports,
+    )
+    container.start()
+    try:
+        yield container
+    finally:
+        container.stop()
+        container.remove()
+        client.images.remove(testee.image_name)
+
+
+def test_jupyterlab(dss_docker_container):
+    """"
+    Test that jupyterlab is configured properly
+    """
+    jupyter_command = (
+        "/root/jupyterenv/bin/jupyter-lab"
+        " --notebook-dir=/root/notebooks"
+        " --no-browser"
+        " --allow-root"
+    )
+    container = dss_docker_container
+    container.exec_run(jupyter_command, detach=True)
+    time.sleep(5.0)
```

Comment:
I would instead poll a certain time the url and fail with a timeout if doesn't come up.